src/app/modules/issue-vc/new-issue-vc/new-issue-vc.component.ts

```diff
@@ -94,7 +98,12 @@ export class NewIssueVcComponent implements OnInit {
       .subscribe(() => this.dialogRef.close());
   }
 
-  private setPossibleRoles() {
+  private setPossibleRoles(): void {
+    this.getPossibleRolesForPassedDID();
+    this.getPossibleRolesWithPredefinedDID();
```

Comment:
When is a DID predefined vs passed?

Reply:
Predefined is when we open this dialog from assets page. You pick asset -> actions -> `Issue Claim to Asset`
Passed is when you are setting manually/by qr code scanner from enrolments page -> Create VC.
Maybe i should use better naming here 🤔

Reply:
Seems okay, thanks for the explaination